.claude/scripts/create_tc_hybrid.py

```
@@ -0,0 +1,190 @@
+        pr_args.extend(['--comment', str(args.comment)])
+
+    pr_draft = run_script('fetch_github_pr.py', pr_args)
+    print(f"      ✓ Test Steps: {len(pr_draft.get('test_steps', []))}", file=sys.stderr)
+    print()
+
+    # Step 3: Merge drafts
+    print("[3/5] Merging Jira + PR data...", file=sys.stderr)
+    merged_draft = merge_jira_pr_drafts(jira_draft, pr_draft)
+
+    # Save merged draft
+    output_dir = '/tmp/test-specs/drafts'
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/test-specs/drafts"

(S108)